model-optimizer/extensions/back/ChangeOutputTypeAttributes.py

```diff
@@ -52,7 +52,7 @@ def find_and_replace_pattern(self, graph: Graph):
                 if node[dst_type] in [np.float32, np.float64] and ir_data_type == np.float16 and \
                         not node.has_and_set('returns_shape_value'):
                     final_type = np.float16
-                elif node.has_and_set('returns_shape_value') and node.dst_type == np.float16:
```

Comment:
I don't agree with this change. The attribute "dst_type" of `Cast` operation and "output_type" of `Range` operation must always exist. Could you analyse why some models failed to convert before your change? What was the reason that the mentioned above attributes were not set?

Reply:
`dst_type` is a string so we should've been referred to the node attributes as `node[dst_type]` not `node.dst_type` which is equivalent to `node."dst_type"/node."output_type"`. @iimironov's changes are correct.

Reply:
dst_type is a string so we should've been referred to the node attributes as node[dst_type] not node.dst_type which is equivalent to node."dst_type"/node."output_type". @iimironov's changes are correct.